packages/core/src/components/modal-content/modal-content.scss

```diff
@@ -15,4 +15,5 @@
   position: relative;
   overflow: auto;
   padding: 0.125rem 0.625rem 0.125rem 0.625rem;
+  max-height: calc(80vh - 116px - 1em);
```

Comment:
Full-size modals will not cover whole screen height with this max-height

Reply:
Removed line because its not necessary.